default.vcl.tmpl

```diff
@@ -355,16 +376,15 @@ sub vcl_synth {
   if (resp.status == 503 && req.http.host != "error") {
     if (resp.http.Follow-Error-Status) {set req.http.Error-Status = resp.http.Follow-Error-Status;}
     else {set req.http.Error-Status = resp.status;}
-    return(restart);
+    return (restart);
   } else if (resp.status == 301) {
     set resp.http.location = resp.reason;
-    set resp.reason = "Moved Permanently";
   } else if (resp.status == 302) {
     set resp.http.location = resp.reason;
-    set resp.reason = "Moved Temporarily";
```

Comment:
Varnish is normalizing the `resp.reason` based on the `resp.status` value. This line didn't have any effect.